yjit/src/codegen.rs

```diff
@@ -5954,6 +5961,10 @@ pub struct CodegenGlobals {
 
     // Methods for generating code for hardcoded (usually C) methods
     method_codegen_table: HashMap<u64, MethodGenFn>,
+
+    /// For implementing exit location tracking
+    yjit_raw_samples: Vec<VALUE>,
+    yjit_line_samples: Vec<i32>
```

Comment:
The state for this should be stored in `stats.rs`, not in `codegen.rs`

Reply:
I wrapped the counter increments in a macro to avoid having to expose the internals of how its accessed.

It just doesn't semantically belong in CodegenGlobals 😅